packages/cli/README.md

```diff
@@ -154,7 +154,7 @@ export class Generator extends Base {
     }    
     protected generateIndexSource(indexFileTargetPath: string) {
         const source = super.generateIndexSource(indexFileTargetPath);
-        return '@namespace "INDEX";\n' + source;
+        return '@st-namespace "INDEX";\n' + source;
```

Comment:
I don't think we want this change in v4. it means that any one would must have the latest version...this is kind of breaking...

Reply:
it's only a docs example...so I'm not sure it matters

Reply:
that's what I thought, it doesn't have any affect on lib usage